src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -77,9 +83,11 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
 
   getDefaultState() {
     const state = super.getDefaultState();
+    const {initialData} = this.props;
```

Comment:
Since initialData would be a `Partial<InviteRow>` we can the just merge each with the invite template

```js
const initialData = this.props.initialData.map(initial => ({...this.inviteTemplate, initial}))
```